mitodl/ocw-studio
# Differentiate between types of user-entered data

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #221 

#### What's this PR do?
- Adds the "content-dir" key to the site config
- Makes it so that if a new WebsiteContent object is for page content (i.e.: the object is being created for a config item that has a file/folder that begins with the "content-dir" value), we set the `is_page_content` flag to `True` for that newly-created object
- Adds a management command for "syncing" the `is_page_content` flag for existing WebsiteContent records

#### How should this be manually tested?
- Create a page content object in the UI (in the OCW course site config, "Page" and "Resource" are configured as page content). The resulting `WebsiteContent` record should have `is_page_content=True`
- Create a non-page content object in the UI (in the OCW course site config, "Metadata" > "Site Metadata" is configured as non-page content). The resulting `WebsiteContent` record should have `is_page_content=False`

You can also test the `sync_contents_with_config` management command to make sure it accurately updates your WebsiteContent records
